app/views/users/shared/_otp_make_default_number.html.erb

```diff
@@ -11,13 +12,14 @@
     <p class="margin-top-0 margin-bottom-2" id="otp_make_default_number_instruction">
       <%= t('two_factor_authentication.otp_make_default_number.instruction') %>
     </p>
-    <label class="btn-border col-8">
-      <div class="checkbox">
-        <%= check_box_tag form_name_label,
-            :otp_make_default_number, form_obj.otp_make_default_number %>
-        <span class="indicator"></span>
-        <%= t('two_factor_authentication.otp_make_default_number.label') %>
-      </div>
+    <%= check_box_tag(
+          form_name_label,
+          :otp_make_default_number,
+          form_obj.otp_make_default_number,
+          class: 'usa-checkbox__input usa-checkbox__input--bordered',
+        ) %>
+    <label class="usa-checkbox__label" for="<%= form_name_tag %>">
```

Comment:
is the `for` here needed for CSS or is it just for accessibility best practices?

Reply:
> is the `for` here needed for CSS or is it just for accessibility best practices?

It's needed both for accessibility and also to maintain the behavior that clicking the label should toggle the checkbox.

Previously we were relying on [implicit label association](https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly) by using the label to wrap both the input and label. That's not an option for the design system checkboxes.